Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManager.cxx

```diff
@@ -553,66 +546,75 @@ bool vtkMRMLMarkupsDisplayableManager::IsCorrectDisplayableManager()
     vtkErrorMacro ("IsCorrectDisplayableManager: No selection node in the scene.");
     return false;
     }
+  const char* placeNodeClassName = selectionNode->GetActivePlaceNodeClassName();
   if (selectionNode->GetActivePlaceNodeClassName() == nullptr)
     {
     return false;
     }
-  // the purpose of the displayableManager is hardcoded
-  return this->IsManageable(selectionNode->GetActivePlaceNodeClassName());
 
+  vtkSmartPointer<vtkMRMLNode> node;
+  node.TakeReference(this->GetMRMLScene()->CreateNodeByClass(placeNodeClassName));
+  vtkMRMLMarkupsNode* markupsNode = vtkMRMLMarkupsNode::SafeDownCast(node);
```

Comment:
A single command would convey the intent a bit more clearly:
vtkSmartPointer<vtkMRMLNode> node = vtkSmartPointer<vtkMRMLNode>::Take(this->GetMRMLScene()->CreateNodeByClass(placeNodeClassName));